src/components/Search.tsx

```diff
@@ -0,0 +1,26 @@
+import { FormEvent, useRef } from "react";
+
+interface Props {
+  initialSearchText: string;
+  onSearch: (searchText: string) => void;
+}
+export default function Search({ initialSearchText, onSearch }: Props) {
+  const inputElement = useRef<HTMLInputElement>(null);
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    onSearch(inputElement.current!.value);
```

Comment:
It make sense to check before onSearch call.

Reply:
Added check for searched value

Reply:
Maybe I was not clear enough.
I was talking about smth like this:
```
if (inputElement.current) {
   onSearch(inputElement.current.value);
}
```